src/Components/PurchaseCard/PurchaseCard.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
+import {
+  Button,
+  Card,
+  CardBody,
+  Flex,
+  Grid,
+  GridItem,
+  Text,
+  TextContent,
+  TextVariants,
+} from '@patternfly/react-core';
+
+const PurchaseCard = () => {
+  const push = useChrome(({ chromeHistory: { push } = {} }) => push);
+
+  return (
+    <Card>
+      <CardBody>
+        <Grid>
+          <GridItem
+            md={6}
+            lg={8}
+            component={TextContent}
+            className="pf-u-pr-2xl"
+          >
+            <Text component={TextVariants.h2} className="pf-u-mb-md">
+              Continue your experience with Red Hat OpenShift service on AWS
+            </Text>
+            <Text
+              component={TextVariants.p}
+              className="pf-u-color-200 pf-u-mb-lg"
+            >
+              Get started with your own dedicated cluster on AWS, managed by Red
+              Hat and Amazon, and even move your application from Dev Sandbox.
+            </Text>
+            <Flex className="pf-u-flex-direction-column pf-u-flex-direction-row-on-md pf-u-mt-lg pf-u-mt-0-on-md">
+              <Button
```

Comment:
The button should have a `component="a"` prop to help with accessibility. And also a `href` attribute.